<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Employés</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QMainWindow { background-color: #F5EBE0; }
    QWidget { background-color: #F5EBE0; color: #3D2817; font-family: 'Segoe UI', Arial, sans-serif; }
    QTabWidget::pane { border: 2px solid #8B5A2B; background-color: #F5EBE0; border-radius: 5px; }
    QTabBar::tab { background-color: #C9B59A; color: #3D2817; padding: 10px 20px; margin-right: 2px; border-top-left-radius: 5px; border-top-right-radius: 5px; font-weight: bold; }
    QTabBar::tab:selected { background-color: #8B5A2B; color: #F5EBE0; }
    QTabBar::tab:hover { background-color: #9B8068; }
    QGroupBox { background-color: #FFFFFF; border: 2px solid #8B5A2B; border-radius: 8px; margin-top: 15px; padding-top: 15px; font-weight: bold; }
    QGroupBox::title { color: #5D4037; subcontrol-origin: margin; left: 10px; padding: 0 5px; font-size: 10pt; }
    QLineEdit, QDoubleSpinBox, QComboBox { background-color: #FFFFFF; border: 2px solid #C9B59A; border-radius: 5px; padding: 8px; color: #3D2817; font-size: 10pt; }
    QLineEdit:focus, QDoubleSpinBox:focus, QComboBox:focus { border: 2px solid #8B5A2B; }
    QComboBox { font-size: 10pt; min-height: 28px; }
    QComboBox::drop-down { border: none; width: 20px; }
    QComboBox::down-arrow { image: none; border-left: 4px solid transparent; border-right: 4px solid transparent; border-top: 4px solid #8B5A2B; margin-right: 6px; }
    QPushButton { background-color: #8B5A2B; color: #F5EBE0; border: none; border-radius: 5px; padding: 10px 20px; font-weight: bold; font-size: 10pt; }
    QPushButton:hover { background-color: #9B8068; }
    QPushButton:pressed { background-color: #5D4037; }
    QPushButton#btnAjouter { background-color: #8B9B6A; }
    QPushButton#btnAjouter:hover { background-color: #9BAB7A; }
    QPushButton#btnModifier { background-color: #8B5A2B; }
    QPushButton#btnModifier:hover { background-color: #9B8068; }
    QPushButton#btnSupprimer { background-color: #5D4037; }
    QPushButton#btnSupprimer:hover { background-color: #6D4C41; }
    QPushButton#btnAnnuler, QPushButton#btnVider, QPushButton#btnAfficherTout, QPushButton#btnTrier, QPushButton#btnRechercher { background-color: #C9B59A; color: #3D2817; }
    QPushButton#btnAnnuler:hover, QPushButton#btnVider:hover, QPushButton#btnAfficherTout:hover, QPushButton#btnTrier:hover, QPushButton#btnRechercher:hover { background-color: #D9C5AA; }
    QTableWidget { background-color: #FFFFFF; alternate-background-color: #F5EBE0; border: 2px solid #C9B59A; border-radius: 5px; gridline-color: #C9B59A; color: #3D2817; }
    QTableWidget::item:selected { background-color: #8B5A2B; color: #F5EBE0; }
    QHeaderView::section { background-color: #8B5A2B; color: #F5EBE0; padding: 6px; border: none; font-weight: bold; font-size: 10pt; }
    QPushButton#btnEmployes { background-color: #C9B59A; color: #3D2817; border: none; border-left: 0px solid #8B5A2B; border-radius: 0; padding: 20px 15px; font-size: 15pt; font-weight: bold; text-align: left; }
    QPushButton#btnEmployes:hover { background-color: #9B8068; color: white; }
    QPushButton#btnEmployes:checked { background-color: #8B5A2B; color: #F5EBE0; border-left: 6px solid #5D4037; }
    QPushButton#btnEmployes:pressed { background-color: #5D4037; color: white; }
   </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayoutPrincipal" stretch="0,1">
    <item>
     <widget class="QWidget" name="sidebar" native="true">
      <property name="minimumWidth">
       <number>180</number>
      </property>
      <property name="maximumWidth">
       <number>180</number>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #C9B59A; border-right: 2px solid #8B5A2B;</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayoutSidebar">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="labelLogo">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>150</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>150</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="logo.qrc">:/images/logo.jpeg</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacerLogoToButton">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnEmployes">
         <property name="text">
          <string>Employés</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacerSidebar">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabConsulter">
       <attribute name="title">
        <string>Consulter</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayoutConsulter">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QGroupBox" name="groupRechercheEtTri">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Recherche et Tri</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalRechercheEtTri">
           <property name="spacing">
            <number>10</number>
           </property>
           <property name="leftMargin">
            <number>10</number>
           </property>
           <property name="topMargin">
            <number>10</number>
           </property>
           <property name="rightMargin">
            <number>10</number>
           </property>
           <property name="bottomMargin">
            <number>10</number>
           </property>
           <item alignment="Qt::AlignVCenter">
            <widget class="QLabel" name="labelRechercher">
             <property name="text">
              <string>Rechercher :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditRecherche">
             <property name="placeholderText">
              <string>CIN ou poste...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnRechercher">
             <property name="text">
              <string>Rechercher</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="spacerRecherche">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item alignment="Qt::AlignVCenter">
            <widget class="QLabel" name="labelTrierPar">
             <property name="text">
              <string>Trier par :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBoxTri">
             <property name="minimumWidth">
              <number>160</number>
             </property>
             <item>
              <property name="text">
               <string>Salaire (croissant)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Salaire (décroissant)</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnTrier">
             <property name="text">
              <string>Trier</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnAfficherTout">
             <property name="text">
              <string>Afficher tout</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBoxInfoEmployes">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Informations des Employés</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayoutInfo">
           <property name="spacing">
            <number>10</number>
           </property>
           <property name="leftMargin">
            <number>12</number>
           </property>
           <property name="topMargin">
            <number>12</number>
           </property>
           <property name="rightMargin">
            <number>12</number>
           </property>
           <property name="bottomMargin">
            <number>12</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalID">
             <item>
              <widget class="QLabel" name="labelID">
               <property name="text">
                <string>ID Employé :</string>
               </property>
               <property name="minimumWidth">
                <number>120</number>
               </property>
               <property name="maximumWidth">
                <number>120</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditID">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>1</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalNom">
             <item>
              <widget class="QLabel" name="labelNom">
               <property name="text">
                <string>Nom :</string>
               </property>
               <property name="minimumWidth">
                <number>120</number>
               </property>
               <property name="maximumWidth">
                <number>120</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditNom">
               <property name="placeholderText">
                <string>Entrez le nom</string>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>1</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalPoste">
             <item>
              <widget class="QLabel" name="labelPoste">
               <property name="text">
                <string>Poste :</string>
               </property>
               <property name="minimumWidth">
                <number>120</number>
               </property>
               <property name="maximumWidth">
                <number>120</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditPoste">
               <property name="placeholderText">
                <string>Entrez le poste</string>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>1</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalSalaire">
             <item>
              <widget class="QLabel" name="labelSalaire">
               <property name="text">
                <string>Salaire :</string>
               </property>
               <property name="minimumWidth">
                <number>120</number>
               </property>
               <property name="maximumWidth">
                <number>120</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="doubleSpinBoxSalaire">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>1</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximum">
                <double>999999.99</double>
               </property>
               <property name="value">
                <number>0.00</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalDispo">
             <item>
              <widget class="QLabel" name="labelDispo">
               <property name="text">
                <string>Disponibilité :</string>
               </property>
               <property name="minimumWidth">
                <number>120</number>
               </property>
               <property name="maximumWidth">
                <number>120</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBoxDispo">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>1</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <item>
                <property name="text">
                 <string>disponible</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>en congé</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>absent</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacerForm">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabAffichage">
       <attribute name="title">
        <string>Affichage</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayoutAffichage">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QGroupBox" name="groupBoxActions">
          <property name="title">
           <string>Actions</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayoutActions">
           <property name="spacing">
            <number>10</number>
           </property>
           <item>
            <widget class="QPushButton" name="btnAjouter">
             <property name="text">
              <string>+ Ajouter</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnModifier">
             <property name="text">
              <string>Modifier</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnSupprimer">
             <property name="text">
              <string>Supprimer</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnAfficherTout">
             <property name="text">
              <string>Afficher Tout</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacerActions">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btnVider">
             <property name="text">
              <string>Vider</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidgetEmployes">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <column>
           <property name="text">
            <string>ID Employé</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Nom</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Poste</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Salaire</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Disponibilité</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBoxExport">
          <property name="title">
           <string>Exporter la Liste des Employés</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayoutExport">
           <item>
            <widget class="QLabel" name="labelExportInfo">
             <property name="text">
              <string>Choisissez le format d'exportation de la liste des employés :</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayoutExportButtons">
             <item>
              <widget class="QPushButton" name="btnExportPDF">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>50</height>
                </size>
               </property>
               <property name="text">
                <string>Exporter en PDF</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBoxExportOptions">
             <property name="title">
              <string>Options d'exportation</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayoutOptions">
              <item>
               <widget class="QCheckBox" name="checkBoxInclureEntete">
                <property name="text">
                 <string>Inclure l'en-tête</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBoxDateExport">
                <property name="text">
                 <string>Inclure la date d'exportation</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabStatistiques">
       <attribute name="title">
        <string>Statistiques</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayoutStats">
        <item>
         <widget class="QGroupBox" name="groupBoxStats">
          <property name="title">
           <string>Statistiques Générales</string>
          </property>
          <layout class="QGridLayout" name="gridLayoutStats">
           <item row="0" column="0">
            <widget class="QLabel" name="labelTotalEmployes">
             <property name="font">
              <font>
               <family>Segoe UI</family>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Nombre total d'employés :</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="labelTotalEmployesValeur">
             <property name="font">
              <font>
               <family>Segoe UI</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="btnActualiserStats">
             <property name="text">
              <string>Actualiser les Statistiques</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacerStats">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="logo.qrc"/>
 </resources>
 <connections/>
</ui>
